content/10-minute-tutorial.md

```diff
@@ -1,3 +1,328 @@
+
+You should see something like the following:
+```
+[INFO] Scanning for projects...
+[INFO]
+[INFO] ------------------------------------------------------------------------
+[INFO] Building cucumber-tutorial 1.0-SNAPSHOT
+[INFO] ------------------------------------------------------------------------
+
+    <progress messages....>
+
+[INFO] ------------------------------------------------------------------------
+[INFO] BUILD SUCCESS
+[INFO] ------------------------------------------------------------------------
+[INFO] Total time: <time> s
+[INFO] Finished at: <date> <time>
+[INFO] Final Memory: <X>M/<Y>M
+[INFO] ------------------------------------------------------------------------
+
+```
+
+Your project builds correctly, but nothing is tested yet as you have not specified any behaviour to test against.
+
+# Specifying Expected Behaviour
+
+We specify the expected behaviour by defining features and scenarios.
+
+## Creating the Feature Directory
+
+Features are defined in feature files, which are stored in the `src/test/resources/` directory (or a subdirectory).
+
+We need to create this directory, as it was not created for us. In IntelliJ:
+
+1. Right click on the Test folder, select **New > Directory** and name this directory `resources`.
+
+2. Right click the folder and select **Mark directory as > Test Resources Root**.
+
+3. You can add subdirectories as needed. Create a subdirectory for your project in `src/test/resources/`
+
+Our project structure is now as follows:
+
+```
+├── pom.xml             (containing our Cucumber and JUnit dependencies)
+└── src
+    ├── main
+    │   └── java        (marked as sources root)
+    │   └── resources   (marked as resources root)
+    └── test
+        ├── java        (marked as test sources root)
+        └── resources   (marked as test resources root)
+                └── <project>
+```
+
+## Creating a Feature
+
+To create a feature file:
+
+1. Open the project in your IDE (if needed) and right-click on the `src/test/resources/<project>` folder.
+
+2. Select **New > File**
+
+3. Enter a name for your feature file, and use the `.feature` extension. For instance, `belly.feature`.
+
+Files in this folder with an extension of `.feature` are automatically recognised as feature files. Each feature file describes a single feature, or part of a feature.
+
+Open the file (double click it) and add the feature description, starting with the `Feature` keyword and an optional description.
+
+For instance:
+```gherkin
+Feature: Belly
+  Optional description of the feature
+```
+
+## Creating a Scenario
+
+Scenarios are added to the feature file, to define examples of the expected behaviour. These scenarios can be used to test the feature.
+Start a scenario with the `Scenario` keyword and add a brief description of the scenario. To define the scenario, you have to define all of its steps.
+
+## Defining Steps
```

Comment:
s/##/###/ ?

Reply:
@plaindocs not sure what you mean here?

Reply:
@mlvandijk Attempting to translate: "Should we use a `###` level of heading here?"

Reply:
thanks @olleolleolle :) Yes, the scenario and steps could be considered part of the feature and a lower level heading

Reply:
Sorry for the non obvious comment, and thanks for the translation :-)